client/components/mma/switch/review/SwitchReview.stories.tsx

```diff
@@ -18,6 +18,7 @@ export default {
 	parameters: {
 		layout: 'fullscreen',
 		reactRouter: {
+			state: { productDetail: contributionPayPal },
```

Comment:
I'm not sure why this is here?

Reply:
![image](https://user-images.githubusercontent.com/114918544/224013672-1cb4275e-33cb-418e-aacb-272715094fd8.png)

The storybook for Switch Review pages looks a bit weird now with two payment methods showing

Reply:
Yes, that `state` shouldn't be there as it's being set at the individual story level due to merging of state causing the above issue. (There's a lengthy comment a bit further down in the file explaining this.)

Reply:
Great spot..I'm not sure how this snuck in here